package.json

```diff
@@ -11,7 +11,7 @@
     "test-unit": "npm run base-test-unit test/unit",
     "test-functional": "mocha -t 60000 --recursive test/functional",
     "test-with-coverage": "nyc -n src -r html cross-env npm run test-unit",
-    "base-test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --require @babel/register",
+    "base-test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --require @babel/register test/unit/bootstrap.js",
```

Comment:
Are you sure it is working correctly ? From what I recall, the `--require` part tries to load iTowns before the next file. So in case iTowns loads first, it will emit an error.

Reply:
yes, it's better to use `--file`  : Explicitly include a test file to be loaded before other test files
https://mochajs.org/#-file-filedirectoryglob

Reply:
Yes, but what I meant is the `--require` is loaded before any other things: "Require a module before loading the user interface or **test files**."
https://mochajs.org/#-require-module-r-module

Reply:
`@babel/register` is required to compile unit tests.